configuration/controller/rsyslog.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+######################################################################
+# TrinityX
+# Copyright (c) 2016  ClusterVision B.V.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License (included with the sources) for more
+# details.
+######################################################################
+
+
+set -ex
```

Comment:
Please don't use `set -e`. See the documentation:
https://github.com/clustervision/trinityX/blob/development/doc/config_post_scripts.rst#shell-script-error-management